CMakeLists.txt

```diff
@@ -361,7 +364,8 @@ if(BUILD_TESTING)
         ${CMAKE_BINARY_DIR}/lib/libgmock.a)
   elseif(WIN32)
     # Make sure we are always bootsrapped with vcpkg on Windows
-    find_package(GTest)
+    # set(CMAKE_FIND_DEBUG_MODE TRUE)
```

Comment:
do we need this?

Reply:
> do we need this?

Thanks for catching, this comment was used for local debugging, removed it.